17/01/14 - 13:50 : System.Windows.Threading.Dispatcher
17/01/14 - 13:51 : System.Windows.Markup.XamlParseException: 'L'invocation du constructeur sur le type 'QuidditchWPF.Login' qui correspond aux contraintes de liaison spécifiées a levé une exception.' numéro de ligne '3' et position de ligne '9'. ---> System.DivideByZeroException: Tentative de division par zéro.
   à QuidditchWPF.Login..ctor() dans c:\Users\Alexandre\Documents\Visual Studio 2012\Projects\QuidditchConsole\QuidditchWPF\Login.xaml.cs:ligne 28
   --- Fin de la trace de la pile d'exception interne ---
   à System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   à System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   à System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   à System.Windows.Application.LoadBamlStreamWithSyncInfo(Stream stream, ParserContext pc)
   à System.Windows.Application.LoadComponent(Uri resourceLocator, Boolean bSkipJournaledProperties)
   à System.Windows.Application.DoStartup()
   à System.Windows.Application.<.ctor>b__1(Object unused)
   à System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   à MS.Internal.Threading.ExceptionFilterHelper.TryCatchWhen(Object source, Delegate method, Object args, Int32 numArgs, Delegate catchHandler)
17/01/14 - 13:51 : System.Windows.Markup.XamlParseException: 'L'invocation du constructeur sur le type 'QuidditchWPF.Login' qui correspond aux contraintes de liaison spécifiées a levé une exception.' numéro de ligne '3' et position de ligne '9'. ---> System.DivideByZeroException: Tentative de division par zéro.
   à QuidditchWPF.Login..ctor() dans c:\Users\Alexandre\Documents\Visual Studio 2012\Projects\QuidditchConsole\QuidditchWPF\Login.xaml.cs:ligne 28
   --- Fin de la trace de la pile d'exception interne ---
   à System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   à System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   à System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   à System.Windows.Application.LoadBamlStreamWithSyncInfo(Stream stream, ParserContext pc)
   à System.Windows.Application.LoadComponent(Uri resourceLocator, Boolean bSkipJournaledProperties)
   à System.Windows.Application.DoStartup()
   à System.Windows.Application.<.ctor>b__1(Object unused)
   à System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   à MS.Internal.Threading.ExceptionFilterHelper.TryCatchWhen(Object source, Delegate method, Object args, Int32 numArgs, Delegate catchHandler)
26/01/14 - 12:34 : System.InvalidOperationException: Une liaison TwoWay ou OneWayToSource ne peut pas fonctionner sur la propriété en lecture seule 'Prenom' de type 'QuidditchWPF.ReservationViewModel'.
   à MS.Internal.Data.PropertyPathWorker.CheckReadOnly(Object item, Object info)
   à MS.Internal.Data.PropertyPathWorker.ReplaceItem(Int32 k, Object newO, Object parent)
   à MS.Internal.Data.PropertyPathWorker.UpdateSourceValueState(Int32 k, ICollectionView collectionView, Object newValue, Boolean isASubPropertyChange)
   à MS.Internal.Data.ClrBindingWorker.AttachDataItem()
   à System.Windows.Data.BindingExpression.Activate(Object item)
   à System.Windows.Data.BindingExpression.AttachToContext(AttachAttempt attempt)
   à System.Windows.Data.BindingExpression.MS.Internal.Data.IDataBindEngineClient.AttachToContext(Boolean lastChance)
   à MS.Internal.Data.DataBindEngine.Task.Run(Boolean lastChance)
   à MS.Internal.Data.DataBindEngine.Run(Object arg)
   à MS.Internal.Data.DataBindEngine.OnLayoutUpdated(Object sender, EventArgs e)
   à System.Windows.ContextLayoutManager.fireLayoutUpdateEvent()
   à System.Windows.ContextLayoutManager.UpdateLayout()
   à System.Windows.ContextLayoutManager.UpdateLayoutCallback(Object arg)
   à System.Windows.Media.MediaContext.InvokeOnRenderCallback.DoWork()
   à System.Windows.Media.MediaContext.FireInvokeOnRenderCallbacks()
   à System.Windows.Media.MediaContext.RenderMessageHandlerCore(Object resizedCompositionTarget)
   à System.Windows.Media.MediaContext.RenderMessageHandler(Object resizedCompositionTarget)
   à System.Windows.Media.MediaContext.Resize(ICompositionTarget resizedCompositionTarget)
   à System.Windows.Interop.HwndTarget.OnResize()
   à System.Windows.Interop.HwndTarget.HandleMessage(WindowMessage msg, IntPtr wparam, IntPtr lparam)
   à System.Windows.Interop.HwndSource.HwndTargetFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   à MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   à MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   à System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   à MS.Internal.Threading.ExceptionFilterHelper.TryCatchWhen(Object source, Delegate method, Object args, Int32 numArgs, Delegate catchHandler)
26/01/14 - 12:34 : System.InvalidOperationException: Une liaison TwoWay ou OneWayToSource ne peut pas fonctionner sur la propriété en lecture seule 'Adresse' de type 'QuidditchWPF.ReservationViewModel'.
   à MS.Internal.Data.PropertyPathWorker.CheckReadOnly(Object item, Object info)
   à MS.Internal.Data.PropertyPathWorker.ReplaceItem(Int32 k, Object newO, Object parent)
   à MS.Internal.Data.PropertyPathWorker.UpdateSourceValueState(Int32 k, ICollectionView collectionView, Object newValue, Boolean isASubPropertyChange)
   à MS.Internal.Data.ClrBindingWorker.AttachDataItem()
   à System.Windows.Data.BindingExpression.Activate(Object item)
   à System.Windows.Data.BindingExpression.AttachToContext(AttachAttempt attempt)
   à System.Windows.Data.BindingExpression.MS.Internal.Data.IDataBindEngineClient.AttachToContext(Boolean lastChance)
   à MS.Internal.Data.DataBindEngine.Task.Run(Boolean lastChance)
   à MS.Internal.Data.DataBindEngine.Run(Object arg)
   à System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   à MS.Internal.Threading.ExceptionFilterHelper.TryCatchWhen(Object source, Delegate method, Object args, Int32 numArgs, Delegate catchHandler)
